src/plugin/utc/util.js

```diff
@@ -0,0 +1,26 @@
+function timezoneOffsetToString(tz) {
+  if (tz === 0) return 'Z'
+  const h = Math.floor(Math.abs(tz) / 60)
+  const m = Math.abs(tz) % 60
+  return `${tz > 0 ? '-' : '+'}${h < 10 ? '0' : ''}${h}${m < 10 ? '0' : ''}${m}`
```

Comment:
Comment about ${h < 10 ? '0' : ''}${h}

Probably function padStart from utils of core code is better? :)
${padStart(h, 2, '0')}

Reply:
thanks, i replaced this method with `padZoneStr` from core code.